crates/runtimelib/src/messaging/mod.rs

```diff
@@ -214,11 +220,22 @@ pub struct JupyterMessage {
     pub parent_header: Option<Header>,
     pub metadata: Value,
     pub content: JupyterMessageContent,
-    #[serde(skip_serializing, skip_deserializing)]
+    #[serde(serialize_with = "serialize_base64", skip_deserializing)]
     pub buffers: Vec<Bytes>,
     pub channel: Option<Channel>,
 }
 
+// Custom serializer for Base64 encoding for buffers
+fn serialize_base64<S>(data: &[Bytes], serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    data.iter()
+        .map(|bytes| BASE64_STANDARD.encode(bytes))
+        .collect::<Vec<_>>()
+        .serialize(serializer)
+}
```

Comment:
We should do this inside of the sidecar.rs setup since it's a bit adhoc that we're converting to base64 (compared to other jupyter clients).